data-prepper-plugins/prometheus-sink/build.gradle

```diff
@@ -0,0 +1,27 @@
+
+dependencies {
+    implementation project(':data-prepper-api')
+    implementation libs.armeria.core
+    implementation project(path: ':data-prepper-plugins:common')
+    implementation 'com.arpnetworking.metrics:prometheus-remote-protocol:1.0.1'
+    implementation project(':data-prepper-plugins:aws-plugin-api')
+    implementation 'io.micrometer:micrometer-core'
+    implementation 'com.fasterxml.jackson.core:jackson-core'
+    implementation 'com.fasterxml.jackson.core:jackson-databind'
+    implementation 'org.apache.commons:commons-compress:1.21'
+    implementation 'joda-time:joda-time:2.11.1'
```

Comment:
Why are you using Joda? We should use Java 8 time whenever possible.

Reply:
Thanks for the review @dlvenable , Fixed